study.md

```diff
@@ -95,11 +106,13 @@ would use to satisfy these requirements.
 In your own words, explain the role of application seed data.
 
 ```md
-<!-- your response here -->
+Seed data put some initial data into a database when it is created. This allows
+the developer to ensure that the database iw working properly
 ```
 
 Should you use seeds to create data to experiment with during development?
 
 ```md
-<!-- your response here -->
+Yes, seeds should be used in rails, as it is far simpler and thus harder
+to mess up than writing a migration to seed a database
 ```
```

Comment:
You answered a different question than the one that was asked. Seed data is *only* for data *required* by your app in order for it work. Experimental data to play with does not meet this criteria.